jenkins/scripts/run_tests.py

```
@@ -0,0 +1,705 @@
+    isolate_count = 0
+
+    if args.render:
+        if not args.test_db_list:
+            print("Error: --test-db-list is required when using --render")
+            sys.exit(1)
+        regular_list, isolate_list, regular_count, isolate_count = render_test_list(
+            args.test_db_list, working_dir, args.splits, args.group, args.perf_mode
+        )
+    else:
+        if regular_list and os.path.exists(regular_list):
+            regular_count = len([l for l in Path(regular_list).read_text().splitlines() if l.strip()])
```

Ruff (0.15.6)

[error] Ambiguous variable name: `l`

(E741)